src/ubx.h

```diff
@@ -49,254 +49,265 @@
 
 #pragma once
 
-#include "gps_helper.h"
-#include "base_station.h"
-#include "../../definitions.h"
```

Comment:
These are required here (at least `#include "base_station.h"`) as the header requires definitions from them. It might compile now, but reodering some includes in the .cpp would lead to compilation failure.

Reply:
Thanks @bkueng .  `#include "base_station.h" was back in place prior to the comment above, and I've now replaced the others you indicated.

Reply:
Sorry, but I don't see this here?